packages/admin/publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+sed -i 's=@airnode/protocol=@api3/airnode-protocol=g' ./src/evm.ts
```

Comment:
`sed` stuff is absolutely disgusting. Let me know if you can think of an alternative way of doing this.

Reply:
Nothing comes to mind